tests/test_decoder.js

```diff
@@ -3,7 +3,7 @@ const path = require('path');
 const { test } = require('uvu');
 const assert = require('uvu/assert');
 
-const { NbsDecoder } = require('..');
```

Comment:
Did this break somehow? `require('..')` should work, since it'll find the entry point from `package.json`, resolving to the `nbsdecoder.js` file.

Reply:
Yeah this still works, I've reverted it. Not sure why I changed it...